pluralsh/plural
# Implement Publisher screen

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/4154003/187690015-f55c809f-a31b-4f32-8418-060f61ebde8a.png">

To test the QA:
- Go to any repo.
- Click on "Plural" on the SideNav
Then:
- Create a publisher from the user menu
- Should redirect to the publisher screen

Also resolves ENG-576

## Summary







## Labels





## Test Plan



## Checklist



- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.
